### PR TITLE
Remove 'OBJECT_ID' from builtin headers.

### PR DESCRIPTION
**Pull request by LaszloLango**
JerryScript-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

File: jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.inc.h [hunk 1]
```diff
@@ -34,7 +29,6 @@
 #endif /* !ROUTINE */
 
 /* Object identifier */
-OBJECT_ID (ECMA_BUILTIN_ID_ARRAY_PROTOTYPE)
```
Comment (zherczeg) [inline jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.inc.h]: Patch looks good, but why do we leave the comment? I don't see the purpose.

Comment (LaszloLango) [inline jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.inc.h]: Sorry, I missed the comments. It was not intentional. :) I'll fix it.

**Comment by LaszloLango** [comment]
Thanks for the reviews. @tilmannOSG, I've updated the commit message. Please check.

**Comment by tilmannOSG** [comment]
@LaszloLango Thanks, looks good!

**Pull request merged**